### ISSUE TITLE
[Telemetry] - Find code locations for counting packet amounts and number of packets sending money over the network

### ISSUE THREAD

**Issue by JoblersTune** [interledger/rafiki#1925]
To count the total and average money [per time frame] over the network we need to access that information from the packets rather than the account information which is private. Need to find where best to fit these metric readings into the code.

**Comment by JoblersTune** [comment]
Packet description for reference:
![image](https://github.com/interledger/rafiki/assets/77615158/7003b820-b43a-4a46-b735-bb09716b25e9)

**Comment by JoblersTune** [comment]
For reference:
[4.3.3. Connection Asset Details](https://interledger.org/rfcs/0029-stream/#:~:text=third%2Dparty%20endpoint.-,4.3.3.%20Connection%20Asset%20Details,exchange%20rates.%20Refer%20to%20Section%203.4%20(Exchange%20Rates)%20for%20more%20information.,-4.4.%20Streams)
Either endpoint MAY expose its asset details by sending a ConnectionAssetDetails frame in a STREAM packet.

Asset details, whether exposed by this frame or obtained by a higher-layer protocol, MUST not change during the lifetime of a Connection. Therefore, if a receiver receives a ConnectionAssetDetails frame that contradicts existing asset details, then the receiver SHOULD close the connection because it would be ambiguous which asset details are authoritative.

While this frame is OPTIONAL, each endpoint SHOULD emit this frame when possible. For example, senders need the recipient to share asset details in order to enforce minimum exchange rates. Refer to Section 3.4 (Exchange Rates) for more information.

**Comment by JoblersTune** [comment]
Questions:

1. Still need to figure out how to access the asset code and scale from the STREAM envelope for these packets so that the amount becomes meaningful.
2. Need to figure out how to make sure we only record the amount once when payment is being made (e.g. not on quote creation and not if packets are replayed).
3. This only triggers for HTTP right? What happens in the context of the admin UI and GraphQL requests for instance?

**Comment by JoblersTune** [comment]
After discussing with @mkurapov we have pinned the location for the amount to be collected in the `post` method of he `createAccountToAccountTransfer` method in `packages/backend/src/accounting/service.ts`.

Currently we have just exposed this in some console.log statements to mark the information and test that we are getting what we expect to get.

We are extracting the amount, asset code and asset scale. However we still need to figure out if we will be converting all the amounts to a single currency (USD) and scale for the purpose of our metrics and display.
